src/main/kotlin/MenuClass.kt

```diff
@@ -1,6 +1,8 @@
 import java.util.Scanner
 open class MenuClass() {
     var menuText = ""
+    private var scanner = Scanner(System.`in`)
```

Comment:
```suggestion
    private val scanner = Scanner(System.`in`)
```
Объект не будет пересоздаваться и ссылке не будет присваиваться новое значение, поэтому val